src/bindings/python/src/pyopenvino/graph/ops/constant.cpp

```diff
@@ -27,6 +27,38 @@ std::vector<size_t> _get_byte_strides(const ov::Shape& s) {
     return byte_strides;
 }
 
+std::vector<size_t> _get_strides(const ov::op::v0::Constant& self) {
```

Comment:
Is it possible to rewrite this into a template?

Reply:
Not really, as the type (assume T in the template) is only known at the runtime after Constant creation. This is why I created this function instead of using `_get_byte_strides`.